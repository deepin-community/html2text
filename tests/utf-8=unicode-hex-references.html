<html>
	<body>
		e&#8209;mailvoorkeuren, wie heeft er geen nodig?
	</body>
</html>
