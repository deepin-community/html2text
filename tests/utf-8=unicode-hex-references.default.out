eâ€‘mailvoorkeuren, wie heeft er geen nodig?
